src/File/FileHelper.php

```diff
@@ -48,6 +48,10 @@ public function normalizePath(string $originalPath): string
 		$path = str_replace('\\', '/', $path);
 		$path = preg_replace('~/{2,}~', '/', $path);
 
+		if ($path === null) {
+			throw new \RuntimeException('preg_replace error: ' . preg_last_error());
```

Comment:
I use custom `ShouldNotHappenException()` for that. Or better, use `Nette\Utils\Strings::replace`.

Reply:
fixed